src/components/Login/Login.module.css

```diff
@@ -0,0 +1,43 @@
+@import url('../../theme.module.css');
+
+.Login {
+    width: 90%;
+    height: 80%;
+    background-color: var(--dark-background-color);
+    padding: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.Login form {
+    margin: 0 auto;
+    max-width: 60%;
+}
+
+.label-input-wrapper {
+    padding-bottom: 4%;
```

Comment:
Defining margins and padding in %s can be buggy, my suggestion is to convert to pixels

Reply:
great suggestion, will make a change